### PR TITLE
Us 05 bottom navigator

### PR DESCRIPTION
**Pull request by giahy1**

### CODE REVIEW

File: lib/src/app.dart [hunk 1]
```diff
@@ -2,11 +2,12 @@ import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 
+import '../../src/screens/navigation_screen.dart';
```
Comment (thanhphamtrung) [inline lib/src/app.dart]: can we change to this?
../src/screens/navigation_screen.dart';

File: test/widget_tests/screens/forgot_password_screen_test.dart [hunk 2]
```diff
@@ -28,10 +27,10 @@ void main() {
     registerFallbackValue(FakeRoute());
   });
 
-  final _forgotPassWordBloc = ForgotPasswordBloc();
+  final _forgotBloc = ForgotPasswordBloc();
```
Comment (thanhphamtrung) [inline test/widget_tests/screens/forgot_password_screen_test.dart]: _forgotBloc => _forgotPasswordBloc

File: test/widget_tests/screens/signup_screen_test.dart [hunk 3]
```diff
@@ -39,12 +41,43 @@ void main() {
...
+    _signupBloc.emit(
+        SignupFailure(emailErrorMessage: AppText.emailMustNotBeEmptyErrorText));
+    await tester.pump();
+    var emailError2 = (tester.widget<EmailTextField>(emailTextField).errorText);
+    print(emailError2);
+    expect(emailError2, AppText.emailMustNotBeEmptyErrorText);
+    _signupBloc.emit(SignupFailure(emailErrorMessage: ''));
+    await tester.pump();
+    var emailError3 = (tester.widget<EmailTextField>(emailTextField).errorText);
+    print(emailError3);
```
Comment (thanhphamtrung) [inline test/widget_tests/screens/signup_screen_test.dart]: remve print

Comment (giahy1) [reply test/widget_tests/screens/signup_screen_test.dart]: done

File: test/widget_tests/screens/forgot_password_screen_test.dart (hunk 2 above)
Comment (giahy1) [reply test/widget_tests/screens/forgot_password_screen_test.dart]: okee

File: lib/src/app.dart (hunk 1 above)
Comment (giahy1) [reply lib/src/app.dart]: i changed it

**Pull request merged**